lib/attributor/types/tempfile.rb

```diff
@@ -31,6 +31,9 @@ def self.load(value,context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
       super
     end
 
+    def self.family
+      String.family
```

Comment:
for this one, I really don't know what the family should be...maybe just "attributor"? ... or "anything"...whatever we want to do for the "Object" equivalent...